File: C#/0674. Longest Continuous Increasing Subsequence.cs
public class Solution {
    public int FindLengthOfLCIS(int[] nums) {
        if (nums.Length==0){
            return 0;
        }
        int rep=0;
        int tmp=1;
        for(int i=1;i<nums.Length;i++){
            if (nums[i]>nums[i-1]){
                tmp+=1;
            }
            else{
                rep=Math.Max(rep,tmp);
                tmp=1;
            }
        }
        return Math.Max(rep,tmp);
    }
}

File: archives/20190519python/0674. Longest Continuous Increasing Subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums: List[int]) -> int:
        if not nums:
            return 0
        rep=0
        cnt=1
        for k in range(len(nums)-1):
            if nums[k+1]>nums[k]:
                cnt+=1
            else:
                rep=max(rep,cnt)
                cnt=1
        return max(rep,cnt)
